doc/sphinx/reference/onedim/nonlinear-solver.md

```diff
@@ -1,5 +1,129 @@
 # Nonlinear Solver for One-dimensional Flows
 
-```{caution}
-This page is a work in progress.
+## Overview
+
+The Newton solver is a numerical method used to find the roots of a nonlinear equation $ F(x) = 0 $. Cantera uses this method for solving the 1-dimensional
+flame equations.
+
+## Method
+
+The function F(x) is a function of the solution vector, x, which is a large vector of all solution components at all grid points.
+
+$$
+\mathbf{x} = \begin{pmatrix}
+u_0 \\
+V_0 \\
+T_0 \\
+\lambda_0 \\
+u_1 \\
+V_1 \\
+T_1 \\
+\vdots
+\end{pmatrix}
+$$
+
+
+### Iteration
+
+The method starts with an initial guess for the solution, $ x^{(0)} $, and iterates until the solution converges.
+
+For each iteration, k, the solution is updated using the following relation:
+
+$$
+J(x^{(k)}) \left( x^{(k+1)} - x^{(k)} \right) = -F(x^{(k)}) \quad (k = 0, 1, 2, 3, \ldots)
+$$
+
+Here, $ \mathbf{J}(x^{(k)}) $ is the Jacobian matrix of $ F(x^{(k)}) $ .
+
+Another way to looking at the equation is:
+
+$$
+ \left( x^{(k+1)} - x^{(k)} \right) = -J(x^{(k)})^{-1} F(x^{(k)})  = \Delta x^{(k)}
+$$
+
+Where $ \Delta x^{(k)} $ is a vector that represents a correction to the current solution that will take the solution
+from $ x^{(k)} $ to $ x^{(k+1)} $. During each iteration this
+correction vector changes and during during the iteration process it should reduce in size until it becomes zero.
+This correction vector points in the direction that will drive the solution towards a zero error.
+
+
+### Jacobian Matrix
+
+The Jacobian matrix $ \mathbf{J} $ is the matrix of partial derivatives:
+
```

Comment:
Yeah, putting the \mathbf{} on every instance of x in the notes seemed like it would create a lot of latex clutter. I might just remove the bold from the early equations to simply everything.